src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { Filter } from '../../services/enums';
+import { TodosContext } from '../../TodosContext';
+import { Todo } from '../../services/types';
+
+function countActiveTodos(todos: Todo[]): number {
+  return todos.filter(todo => {
+    return !todo.completed;
+  }).length;
+}
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos,
+    handleClearAllCompleted,
+    filterBy,
+    setFilterBy,
+    isTodosHasCompleted,
+  } = useContext(TodosContext);
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${countActiveTodos(todos)} `}
+        items left
+      </span>
+
+      <ul className="filters" data-cy="todosFilter">
+        <li>
```

Comment:
can you check some similar things in this link items, create some array, and iterate through it?
like
```
const VALUES = [{ path: '#/', filterValue: FilterField.all, label: 'All' }]


jsx
{VALUES.map(item => (
  <li key={item.path}>
    <a
      href={item.path}
      className={classNames({
        selected: filterField === item.filterValue,
      })}
      onClick={(event) => handleClick(event, item.filterValue)}
    >
      {item.label}
    </a>
  </li>
))}
```

follow DRY principle